os/include/stdlib.h

```diff
@@ -115,11 +115,15 @@ struct mallinfo {
 	int arena;					/* This is the total size of memory allocated
 								 * for use by malloc in bytes. */
 	int ordblks;				/* This is the number of free (not in use) chunks */
-	int mxordblk;				/* Size of the largest free (not in use) chunk */
+	int mxordblk;				/* Size of the largest free (not in use) chunk
+								 *   NOTE : The available size to allocate is smaller than
+ 								 *    the return of this API, because of the size of alloc node.
+ 								 *   available size = free size - SIZEOF_MM_ALLOCNODE*/
```

Comment:
how about largest free size instead of free size? Because there is free size for another value.

Reply:
Modified. Thanks.